NETProvider/src/FirebirdSql.Data.UnitTests/TestsBase.cs

```diff
@@ -31,9 +31,45 @@
 using NUnit.Framework;
 using FirebirdSql.Data.FirebirdClient;
 using FirebirdSql.Data.Services;
+using System.Diagnostics;
 
 namespace FirebirdSql.Data.UnitTests
 {
+	[SetUpFixture]
+	public class FirstClass
+	{
+		[SetUp]
+		public void SetupDatabase()
+		{
+			Prepare(FbServerType.Default);
+			Prepare(FbServerType.Embedded);
```

Comment:
Umm. This fails if you're trying to run just one test in `Default`. I think I need to come up with some on-demand initialization.